README.md

```diff
@@ -1 +1,96 @@
-https://hub.docker.com/r/microsoft/oms/
+# Trying the container solution pack for Microsoft Operations Management Suite
+
+## What is the Microsoft Operations Management Suite?
+
+The Microsoft Operations Management Suite (OMS) is a software-as-a-service offering from Microsoft that allows Enterprise IT to manage any hybrid cloud. It offers log analytics, automation, backup and recovery, and security and compliance.  Sign up for a free account at [http://mms.microsoft.com](http://mms.microsoft.com) or read more about here: [https://www.microsoft.com/en-us/server-cloud/operations-management-suite/overview.aspx](https://www.microsoft.com/en-us/server-cloud/operations-management-suite/overview.aspx)
+
+This container solution pack will generate a container which will runs OMS agent within. This is for Linux OS which has restriction in installing the Operations Management Suite Agent directly. However, it can also be used with other support Linux OS as well.
```

Comment:
Suggest line wrapping this to shorter lines.  Something like:

```
This container solution pack will generate a container which will
runs OMS agent within. This is for Linux OS which has restriction
in installing the Operations Management Suite Agent directly.
However, it can also be used with other support Linux OS as well.
```

These are shorter lines so, in the future, should there be changes, it's easier to identify them.

Also, wording on the above is off ... "... generate a container which will runs OMS ..."